LibreNMS/IRCBot.php

```diff
@@ -162,6 +163,11 @@ private function init()
             }
 
             if (isset($this->tempnick)) {
+                if ($this->nickwait > 100) {
```

Comment:
I don't see what this adds? It just tries twice per tick every 100 loop?

Reply:
Yeah. Just removed one of the lines setting the nick.  
The point of the patch was to try again but not on _every_ tick.  Was a bit of a dance with different branches and patches after tonights update to latest build failed, and this got caught up in the process.